app/docs/0.1.1beta-2/cli.md

```diff
@@ -0,0 +1,132 @@
+---
+title: CLI Reference
+---
+
+# CLI Reference
+
+Kong comes with a ***Command Line Interface*** *(CLI)* which lets you perform operations such as starting and stopping Kong. Each command is run in the context of a single node, since Kong has no cluster awareness yet.
+
+Almost every CLI command requires access to your configuration file in order to be aware of where the NGINX working directory is located (known as the "prefix path" for those familiar with NGINX), and referenced as `nginx_working_dir` in the Kong configuration file.
+
+**Note:** If you haven't already, we recommand you to read the [configuration guide][configuration-guide].
+
+---
+
+## kong
+
+The CLI's entry point is the `kong` command. View the usage info by typing:
+
+```bash
+
+$ kong --help
+```
+
+Like many other CLI tools, the second parameter will be the command you want to perform:
+
+```bash
+$ kong <command>
+```
+
+To print the usage info of any command, execute:
+
+```bash
+$ kong <command> --help
+```
+
+---
+
+## start
+
+The first command you are likely to execute will be `start`. As you would expect, it starts a Kong node on your current machine using a configuration file.
```

Comment:
- Being more direct would be better, CLI reference is less of a guide and more of a reference.

  Initializes a Kong (instance / node), optionally a configuration file may be passed:

  ```bash
  $ kong start [options]
  ```